shared/actions/team-building.tsx

```diff
@@ -8,7 +9,17 @@ import * as RPCTypes from '../constants/types/rpc-gen'
 import {TypedState} from '../constants/reducer'
 import {validateEmailAddress} from '../util/email-address'
 
-const closeTeamBuilding = () => RouteTreeGen.createNavigateUp()
```

Comment:
this naively navigated up without any context , now it checks if its actually a modal still